Makefile

```diff
@@ -77,6 +79,9 @@ check:
 	$(GO) get github.com/AlekSi/gocov-xml
 	gocov convert coverage-*.txt | gocov-xml -source $(shell pwd) > coverage.xml
 	rm -rf coverage-*.txt 
+else
+	$(info Please start redis server to run unit test)
```

Comment:
This will be treated as test passed? Is it expected?

Reply:
When we create a PR for sonic-gnmi repo, azure pipeline will start redis server, and we can run unit test.
But when we use sonic-buildimage to build, there's no redis server, and we should not run unit test.